src/Client.js

```diff
@@ -2,26 +2,23 @@ import dnode from 'dnode';
 import { CLIEngine } from 'eslint';
 import { clearLine } from './cliUtils';
 
-const eslint = new CLIEngine();
-
-function prettyPrintResults(results) {
-  const formatter = eslint.getFormatter();
-  console.log(formatter(results));
-}
-
 export default class Client {
-  constructor(port) {
+  constructor(options) {
+    const { port, json } = options;
+    const eslint = new CLIEngine();
     this.port = port;
+    this.formatter = json ? eslint.getFormatter('json') : eslint.getFormatter();
```

Comment:
true, though ultimately we should be able to pass `formatter` straight to `eslint.getFormatter()` to allow more than just json